cli/src/test/java/hudson/cli/PrivateKeyProviderTest.java

```diff
@@ -22,7 +22,15 @@
     @Test
     public void loadKeyDSA() throws IOException, GeneralSecurityException {
         File file = new File(this.getClass().getResource("dsa").getFile());
-        String password = null;
+        assertkeyPair(file, null);
+    }
+
+    /**
+     * Extracted method to assert the keyPair private and public are not null.
+     * @param file the file to load the key from
+     * @param password the password
+     */
+    private void assertkeyPair(File file, String password) throws IOException, GeneralSecurityException {
```

Comment:
Naming consistency nit:

```suggestion
    private void assertKeyPair(File file, String password) throws IOException, GeneralSecurityException {
```